alembic/versions/007_rename_market_data_to_legacy.py

```
@@ -0,0 +1,81 @@
+    conn = op.get_bind()
+    inspector = sa.inspect(conn)
+
+    if 'market_data' in inspector.get_table_names():
+        # Rename table from market_data to legacy_market_data
+        op.rename_table('market_data', 'legacy_market_data')
+
+        # Rename indexes to match new table name
+        try:
+            op.drop_index('idx_market_data_symbol_timestamp', table_name='legacy_market_data')
+            op.drop_index('idx_market_data_exchange_timestamp', table_name='legacy_market_data')
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)